lib/utils/report.js

```diff
@@ -83,7 +81,11 @@ module.exports = function (violation) {
 		}
 	}
 
-	const severity = _.get(result.stylelint, ['ruleSeverities', ruleName]);
+	if (!result.stylelint.ruleSeverities) {
+		result.stylelint.ruleSeverities = {};
+	}
+
+	const severity = result.stylelint.ruleSeverities[ruleName];
```

Comment:
[suggestion] IMO, it seems safe to avoid the side-effect `result.stylelint.ruleSeverities = {}` here. (I'm not sure if `result.stylelint.ruleSeverities` is falsy or not, though)

How about this? 👇🏼 

```suggestion
	const severity = result.stylelint.ruleSeverities && result.stylelint.ruleSeverities[ruleName];
```

Reply:
I think I was trying to shorten the line length here by setting the property, but I agree it's better to use code that doesn't add a side effect.